### PR TITLE
Feature/test tools

### PR DESCRIPTION
**Pull request by lpil**

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
_If you're new, read the [guidelines](https://github.com/reevoo/guidelines/blob/master/pull_requests.md)._

Good code is **tested**, **designed for change**, **easy to understand** and **able to cope when
things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [ ] Description explains what you have done and why.
- [ ] Tests have been written (or you have explained why you don't need
  tests).
- [x] Documentation/comments have been added/updated.

**For the Reviewer**
- [ ] The code is well-designed and able to cope with change.
- [ ] The code is easy to understand.
- [ ] Tests cover happy and unhappy paths in the code.
- [ ] Tests are green on CI (or pulled/tested on your machine).

File: mix.exs [hunk 1]
```diff
@@ -28,6 +28,11 @@ defmodule Openreevoo.Mixfile do
   #
   # Type `mix help deps` for more examples and options
   defp deps do
-    []
+    [
+      # Automatic test runner
+      {:mix_test_watch, only: :dev},
```
Comment (jonnyarnold) [inline mix.exs:33]: Do these tasks get run automatically, or is there a special command to use?

(If there's a special command, can you put it in the readme?)

Comment (lpil) [inline mix.exs:33]: Sure thing.

**Comment by reevoo-samuel** [comment]
I don't see any comments on your Pull Request.

Are you too good for code reviews now?
No. No you aren't.

Maybe you forgot to say who reviewed it. That's fine, but let me know.

**Comment by jonnyarnold** [comment]
Sorry Samuel...

**Pull request merged**